### DISCUSSION TITLE
Other Solvers than CBC and Gurobi

### DISCUSSION THREAD

**Discussion by ald0025** [coin-or/python-mip#318] [Q&A]
What is the status on incorporating other solvers like CPLEX and SCIP?

**Comment by rschwarz** [comment]
I have an interest in supporting both HiGHS and SCIP, but won't promise any delivery dates.

**Reply by ald0025** [reply]
Thank you for the response. Can you provide an outlook?

**Reply by rschwarz** [reply]
No, it depends on the workload next quarter, which is not known yet.

**Reply by ald0025** [reply]
My ask was to make a distinction of the appropriate timeline, whether it will be worked on in the near future or distant future.

**Reply by sebheger** [reply]
@ald0025 The status is: Nothing happened and nothing is planned in the near or distant future.
